src/mame/drivers/dragon.cpp

```diff
@@ -445,8 +443,8 @@ void dragon_alpha_state::dgnalpha(machine_config &config)
 	// pia 2
 	pia6821_device &pia2(PIA6821(config, PIA2_TAG, 0));
 	pia2.writepa_handler().set(FUNC(dragon_alpha_state::pia2_pa_w));
-	pia2.irqa_handler().set(FUNC(dragon_alpha_state::pia2_firq_a));
-	pia2.irqb_handler().set(FUNC(dragon_alpha_state::pia2_firq_b));
+	pia2.irqa_handler().set(m_firqs, FUNC(input_merger_device::in_w<0>));
+	pia2.irqb_handler().set(m_firqs, FUNC(input_merger_device::in_w<1>));
```

Comment:
Won’t this clash with `pia` from `dragon_base`?  I would’ve thought this one would use `<2>` and `<3>`.  Am I missing something?

Reply:
You are correct. It should be 2 and 3 here. I slowed down and re-read Phill's notes and discovered the NMI line now needs a merge as well (cartridge port and internal FDC). So this will take a bit longer.

Reply:
I'm still working on this...

Reply:
OK. I'm ready to restart the review.